lighthouse-core/config/config.js

```diff
@@ -297,10 +297,12 @@ function deepCloneConfigJson(json) {
   return cloned;
 }
 
+/**
+ * @implements {LH.Config.Json}
```

Comment:
Oh, yeah, and `@implements` works now! No more implements hacks (see deletion below :)